From feb854be2c431083b16593efddf536c3d0e3eaa4 Mon Sep 17 00:00:00 2001
From: Bai Ping <ping.bai@nxp.com>
Date: Tue, 11 Oct 2016 17:09:35 +0800
Subject: [PATCH 1178/5242] MLK-13310-02 ARM: dts: imx: Correct the clock node
 property on imx6sll

commit  3b0ff335aabf4c2aeef50c6c1d97de536bbe473b from
https://source.codeaurora.org/external/imx/linux-imx.git

Add more properties to the clock node.

Signed-off-by: Bai Ping <ping.bai@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6sll.dtsi |    6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/imx6sll.dtsi b/arch/arm/boot/dts/imx6sll.dtsi
index e64f942..a93f875 100644
--- a/arch/arm/boot/dts/imx6sll.dtsi
+++ b/arch/arm/boot/dts/imx6sll.dtsi
@@ -90,14 +90,18 @@
 
 		ckil: clock@0 {
 			compatible = "fixed-clock";
+			reg = <0>;
 			#clock-cells = <0>;
 			clock-frequency = <32768>;
+			clock-output-names = "ckil";
 		};
 
-		osc: osc@1 {
+		osc: clock@1 {
 			compatible = "fixed-clock";
+			reg = <1>;
 			#clock-cells = <0>;
 			clock-frequency = <24000000>;
+			clock-output-names = "osc";
 		};
 
 		ipp_di0: clock@2 {
-- 
1.7.9.5

